cinema/serializers.py

```diff
@@ -69,6 +97,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
The `movie_image` field is defined but not included in the `fields` list of `MovieSessionListSerializer`. Ensure it is added to the `fields` list to be serialized correctly.